Makefile

```diff
@@ -29,6 +29,11 @@ tag: ## Tag the current release
 publish: ## Put the release on GitHub
 	dune-release publish distrib
 
+.PHONY: promote-ppx-output
+promote-ppx-output: ## Promotes the current output of the ppx unit tests to be the new expected output.
+	dune runtest --force; true # Without the 'true' Make will abort.
```

Comment:
Wouldn't `| true` be the more common idiom?

Reply:
In this case it doesn't make a difference, but you're right that `||` (two pipes) is indeed more idiomatic in Bash, and I've therefore made the change.